sdk/communication/Azure.Communication.Identity/CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 # Release History
 
-## 1.4.0-beta.1 (Unreleased)
+## 1.4.0 (Unreleased)
```

Comment:
I think what we're intending to release needs to be the top-most item. Once released, a PR will be automatically generated to add a newer version to the changelog